package.json

```diff
@@ -63,6 +63,7 @@
     "merkle-patricia-tree": "^2.3.2",
     "rustbn.js": "~0.2.0",
     "safe-buffer": "^5.1.1",
+    "tap-xunit": "^2.4.1",
```

Comment:
Is there a way we could have this as a `devDependency`?

Reply:
absolutely, my mistake